docker/Dockerfile

```diff
@@ -198,34 +199,7 @@ RUN wget https://salsa.debian.org/klausenbusk-guest/debootstrap/commit/a9a603b17
     dpkg-buildpackage -us -uc && \
     sudo dpkg -i ../debootstrap*.deb
 
-# Packages needed for Linux Kernel
-# cmake required by accel-ppp
-RUN apt-get update && apt-get install -y \
-      cmake \
-      gnupg2 \
-      rsync \
-      libelf-dev \
-      libncurses5-dev \
-      flex \
-      bison \
-      bc \
-      kmod \
-      cpio \
-      python-is-python3 \
-      dwarves \
-      nasm \
-      rdfind
-
-# Packages needed for Intel QAT out-of-tree drivers
 # FPM is used when generation Debian pckages for e.g. Intel QAT drivers
-RUN apt-get update && apt-get install -y \
-      pciutils \
-      yasm \
-      ruby \
-      libudev-dev \
-      ruby-dev \
-      rubygems \
-      build-essential
```

Comment:
As I understand it, it is used not only for the packages in the `linux-kernel` build
Possible affected packages:
- vyos/vpp
- vyos/vyatta-cfg
- vyos/libpam-radius-auth
- vyos/libnss-mapuser
- vyos/vyatta-bash

and so on... where C is used.